src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -23,12 +23,16 @@
 
   </Target>
 
+  <PropertyGroup>
+      <VersionFeature21>$([MSBuild]::Add($(VersionFeature), 25))</VersionFeature21>
```

Comment:
so 3.1.12 and 2.1.25 (the latest already released version) will be used for 5.0.200

@wtgodbe that is correct?

Reply:
Yes, for implicit versions - and for 5.0.201 it should be 3.1.13 and 2.1.26